### PR TITLE
add tar.gz packaging script and packages

### PR DESCRIPTION
**Pull request by boneyard93501**

### CODE REVIEW

File: aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+mkdir -p service-packages/modules/eth_rpc
+mkdir -p service-packages/modules/ipfs_adapter
+mkdir -p service-packages/modules/ipfs_cli
+mkdir -p service-packages/modules/multi_provider_query
+mkdir -p service-packages/modules/simple_quorum
+mkdir -p service-packages/modules/utilities
+mkdir -p service-packages/modules/curl_adapter
+# sh ./scripts/build.sh
```
Comment (folex) [inline aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh:8]: remove that line?

File: aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+rm service.yaml
+
+# create utility service package
+cp ../configs/services/utilities/service.yaml .
+cp ../artifacts/utilities.wasm modules/utilities/
+cp ../wasm-modules/utilities/module.yaml modules/utilities/
+tar -czf utilities_package.tar.gz service.yaml 
+rm service.yaml
+
+rm -r modules
```
Comment (folex) [inline aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh:51]: modules won't be empty, I think, so you need `rm -rf modules`

File: aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+cd ../
```
Comment (folex) [inline aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh:52]: No need to do that, CWD is changed only inside the shell script.

**Review by folex — COMMENTED** [review]
In the future, I would put these .tar.gz files to Releases

File: aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh (hunk 1 above)
Comment (boneyard93501) [reply aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh:8]: uncomment

File: aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh (hunk 2 above)
Comment (boneyard93501) [reply aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh:51]: rm -r is for non-empty dirs, rm -rf as well but ignore prompts for write-protected dir

File: aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh (hunk 3 above)
Comment (boneyard93501) [reply aqua-examples/decentralized-blockchain-gateway/scripts/package_services.sh:52]: 👍

**Pull request merged**